src/gluonts/torch/model/estimator.py

```diff
@@ -154,7 +154,7 @@ def train_model(
 
         trainer.fit(
             model=training_network,
-            train_dataloader=training_data_loader,
+            train_dataloaders=training_data_loader,
```

Comment:
This used to be different in PL 1.3.1, see https://pytorch-lightning.readthedocs.io/en/1.3.1/api/pytorch_lightning.trainer.trainer.html#pytorch_lightning.trainer.trainer.Trainer.fit

So apparently PL 1.4.0 introduced a breaking change, see https://pytorch-lightning.readthedocs.io/en/1.4.0/api/pytorch_lightning.trainer.trainer.html#pytorch_lightning.trainer.trainer.Trainer.fit

We should probably update the requirement on PL here, too? https://github.com/awslabs/gluon-ts/blob/master/requirements/requirements-pytorch.txt